### ISSUE TITLE
Intellisense is obtrusive and obscures the program while visible

### ISSUE THREAD

**Issue by TeamDman** [TeamDman/SuperFactoryManager#547]
The positioning of the intellisense often hides the program and makes it annoying to keep enabled.
A better mechanism should be implemented that minimizes the amount of the source code text that is obscured by the intellisense visual
